cli/cli.py

```diff
@@ -99,10 +100,6 @@ def main():
         logger.log(logging.INFO, {"event": "wiji.cli.main", "stage": "end"})
 
 
-from multiprocessing import Process
-from threading import Thread
-
-
 def start_loop(coro):
```

Comment:
when using multi-processing, I do not think we need to; 
```python
loop = asyncio.new_event_loop()
asyncio.set_event_loop(loop)
```

I think we can just do;
```python
asyncio.run( per_process_main())
```